<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContainerEditor</class>
 <widget class="QDialog" name="ContainerEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Container Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Container No:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtContainerNo"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cboType">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>40 FT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20 FT</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Notes:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="txtNotes"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="grpReels">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Reels In this Container</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="CustomQueryView" name="lstReels"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="cmdAddReel">
          <property name="text">
           <string>Add Reel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmdRemoveReel">
          <property name="text">
           <string>Remove Reel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cmdPostToReelStock">
          <property name="text">
           <string>Post To Reel Stock</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cmdSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomQueryView</class>
   <extends>QTreeView</extends>
   <header>customqueryview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cmdCancel</sender>
   <signal>clicked()</signal>
   <receiver>ContainerEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>448</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
